homeassistant/components/plugwise/gateway.py

```diff
@@ -92,6 +94,16 @@ def async_migrate_entity_entry(entry: RegistryEntry) -> dict[str, Any] | None:
     """
     if entry.domain == SWITCH_DOMAIN and entry.unique_id.endswith("-plug"):
         return {"new_unique_id": entry.unique_id.replace("-plug", "-relay")}
+    if entry.domain == BINARY_SENSOR_DOMAIN and "auxiliary" in entry.unique_id:
+        return {"new_unique_id": entry.unique_id.replace("-auxiliary", "-opentherm")}
+    if entry.domain == SENSOR_DOMAIN and entry.unique_id.endswith(
+        "-outdoor_temperature"
+    ):
```

Comment:
But `outdoor-temperature` still exists right?

So it should only migrate if `outdoor_temperature` isn't provided by the current setup and does provide `outdoor_air_temperature`, right? 

Because this will cause `outdoor_temperature` to be constantly migrated?